app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  def update_likes_counter
+    post.update(likes_counter: Like.where(post_id: post).count)
+  end
```

Comment:
- [OPTIONAL] As discussed in the above OPTIONAL suggestion, you can use the same idea to refactor this method too.
```suggestion
  def update_likes_counter
    post.increment!(:likes_counter)
  end
```

Reply:
Well solved